services/ai-navigator-app/0.2.0/helmrelease/helmrelease.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: helm.toolkit.fluxcd.io/v2beta1
 kind: HelmRelease
 metadata:
-  name: ai-navigator-cluster-info-api
+  name: ai-navigator-app
```

Comment:
```suggestion
  name: ai-navigator-app-helmrelease
```

Reply:
Will this work? I thought this was the one that had to match the app name in order for the CM to apply correctly.

Reply:
oops sorry you're right! just the flux kust name not this one 🤦🏻‍♀️